### PR TITLE
Add kops manifest

### PR DESCRIPTION
**Pull request by maver1ck**
Signed-off-by: Maciej Bryński <maciek@brynski.pl>

Kubernetes Operations (kops) - Production Grade K8s Installation, Upgrades, and Management

### CODE REVIEW

File: bucket/kops.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+        }
+    },
+    "bin": "kops.exe",
+    "checkver": "github",
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/kubernetes/kops/releases/download/$version/kops-windows-amd64#/kops.exe"
+            }
+        }
```
Comment (Ash258) [inline bucket/kops.json]: Add hash extraction

```suggestion
        },
        "hash": {
            "url": "$url-sha1"
        }
```

File: bucket/kops.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": "1.11.0",
+    "description": "Kubernetes Operations (kops) - Production Grade K8s Installation, Upgrades, and Management.",
+    "homepage": "https://github.com/kubernetes/kops/",
+    "license": "Apache License 2.0",
```
Comment (Ash258) [inline bucket/kops.json]: Invalid SPDX identifier

```suggestion
    "license": "Apache-2.0",
```

File: bucket/kops.json (hunk 1 above)
Comment (Ash258) [reply bucket/kops.json]: Run checkver with Force parameter afterward

Comment (Ash258) [reply bucket/kops.json]: Go to root of repository and execute `.\bin\checkver.ps1 kops -f`

Comment (maver1ck) [reply bucket/kops.json]: Done.

Comment (maver1ck) [reply bucket/kops.json]: Done.

Comment (Ash258) [reply bucket/kops.json]: What's the reason you haven't used `$url` variable?? 

Proof

File: bucket/kops.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{
+    "version": "1.11.0",
+    "description": "Kubernetes Operations (kops) - Production Grade K8s Installation, Upgrades, and Management.",
+    "homepage": "https://github.com/kubernetes/kops/",
+    "license": "Apache-2.0",
```
Comment (Ash258) [inline bucket/kops.json]: https://github.com/kubernetes/kops#prerequisite

> kubectl is required

```suggestion
    "license": "Apache-2.0",
    "depends": "kubectl",
```

**Pull request merged**